index.html

```diff
@@ -62,25 +62,24 @@ <h2 class="presentation">Tonic</h2>
             <li class="counter"><img src="./css/Counter.png" alt=""></li>
             <li class="counter">2015</li>
         </ul>    
-    <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+     <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
         <ul class="list_middle">
             <li class="languaje">html</li> 
             <li class="languaje">css</li>
             <li class="languaje">javascript</li>
         </ul>
-       
          <div class="project">
-            <button  type="submit" class="button_project" onclick="window.location.href='https://github.com/Alondra87/Portfolio1'">See Project</button>
+            <button  class="button_project" onClick="openModal(0)" type="button">See Project</button> <!--  data-modal-target="#modal" -->
         </div>
       </div>
     </div>
-    <div class="middle-reverse">
+    <div class="middle-reverse" id="modal2">
         <div class="mds12">  
         <figure>
             <img class="snaps" src="./css/SnapshootPortfolio2.svg" alt="Snapshoot_Portfolio">
         </figure>
         </div>
-        <div class="mds22">
+    <div class="mds22">
         <h2 class="presentation">Multi-Post Stories</h2>
            <ul class="list_middle">
                 <li class="canopy">CANOPY</li>
```

Comment:
- [ ] Kindly as asked by the requirements the `projects` section should be created dynamically in your js file, so you have to delete all the hard code from the HTML file and create this section using JS. `You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.`